hexrd/ui/mask_regions_dialog.py

```diff
@@ -86,11 +88,8 @@ def create_patch(self):
             'fill': False,
             'animated': True,
         }
-        if self.selection == 'Rectangle':
-            self.patch = patches.Rectangle((0, 0), 0, 0, **kwargs)
-        elif self.selection == 'Ellipse':
-            self.patch = patches.Ellipse((0, 0), 0, 0, **kwargs)
-        self.axes.add_patch(self.patch)
+        self.patch = InteractiveTemplate(self.parent, self.det)
```

Comment:
Should we rename `self.patch` to something else, since it's actually an interactive template and not a patch?

Reply:
Yeah, that makes sense! I think the simplest is best: `self.patch` -> `self.interactive_template`, or we can just use `self.it` like we do in the LLNL import tool. Any preference?

Reply:
Yup, updated.